packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -11,20 +20,93 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
   const style = useStyle();
 
   return (
-    <div class={style('notificationContainer', 'nt-w-full nt-p-6')}>
-      <div class={style('notificationSubjectContainer', 'nt-flex nt-items-center nt-gap-2 nt-relative')}>
-        <span
+    <div
+      class={style(
+        'notification',
+        'nt-w-full nt-text-sm hover:nt-bg-neutral-100 nt-group nt-relative nt-flex nt-px-6 nt-py-4 nt-gap-2'
+      )}
+    >
+      <span
+        class={style(
+          'notificationDot',
+          'nt-absolute -nt-translate-x-[150%] nt-translate-y-1/2 nt-size-2.5 nt-bg-primary nt-rounded-full nt-border'
+        )}
+      />
+      <Show when={props.notification.avatar}>
+        <img class={style('notificationImage', 'nt-size-8 nt-rounded-lg')} src={props.notification.avatar} />
+      </Show>
+      <div class={style('notificationBody', 'nt-overflow-hidden nt-w-full')}>
+        <p
           class={style(
-            'notificationDot',
-            'nt-absolute -nt-left-3 nt-top-0 -nt-translate-x-1/2 nt-translate-y-1/2 nt-size-3 nt-bg-primary nt-rounded-full nt-border'
+            'notificationDate',
+            'nt-text-foreground-alpha-400 nt-shrink-0 nt-float-right group-hover:nt-hidden'
           )}
-        />
+        >
+          {/* TODO: pass locale here */}
+          {formatToRelativeTime({ fromDate: new Date(props.notification.createdAt) })}
+        </p>
+        <div
+          class={style(
+            'notificationDefaultActions',
+            'nt-gap-3 nt-shrink-0 nt-float-right nt-hidden group-hover:nt-flex'
```

Comment:
✏ **suggestion:** Please give a little bit more space from the left side of this container. In case of a long subject the buttons are too close to the text.

Reply:
![Screenshot 2024-07-30 at 12 28 07](https://github.com/user-attachments/assets/9be4ddc6-dcde-4366-b1d6-55c35640a82d)